csrankings-s.csv

```diff
@@ -1147,6 +1147,7 @@ Simeon C. Ntafos,University of Texas at Dallas,http://www.utdallas.edu/~ntafos,N
 Simeon Retalis,University of Piraeus,https://www.ds.unipi.gr/en/faculty/retal-en,JUX2KJcAAAAJ
 Simha Sethumadhavan,Columbia University,http://www.cs.columbia.edu/~simha,NOSCHOLARPAGE
 Simina Brânzei,Purdue University,https://www.cs.purdue.edu/people/faculty/simina,SJYC6DQAAAAJ
+Siming Chen 0001,Fudan University,http://simingchen.me/,LikgkYwAAAAJ

```

Comment:
Please explain how the School of Data Science relates to the CS Department.

Reply:
Our school of Data Science hires teachers from three research focuses, including computer science, statistics, and related applications. There are three majors of students, computer science, statistics, and techniques for finance. Prof. Siming is from the pure computer science part and hires students of computer science majors. His research is data visualization. His colleagues, like Yanwei Fu (CV), and Zhongyu Wei (NLP), who are in the same situation, are all in the CSRankings. Therefore, I believe it is reasonable to add his profile there. Thanks!

Reply:
So please clarify: can this faculty member solely advise PhD students for a degree _in Computer Science_?

Reply:
Yes, he can.